staramr/SubCommand.py

```diff
@@ -34,7 +35,9 @@ def run(self, args):
         :return: None
         """
         if args.verbose:
+            coloredlogs.install(level='DEBUG')
```

Comment:
It looks like `coloredlogs` modifies the format of the log output beyond just the color. For example:

### Before coloredlogs (with `--verbose`)

```
2019-04-17 16:00:35,340 INFO BlastHandler._make_db_from_input_files,101: Making BLAST databases for input files
```

### After coloredlogs (with `--verbose`)

```
2019-04-17 16:01:33 phac5021054 BlastHandler[17903] INFO Making BLAST databases for input files
```

That is, before it was printing the function name and line number `BlastHandler._make_db_from_input_files,101` in debug mode. But afterwards, these are not printed.

Could you maybe take a look into if it's possible to change the format to at least include the line number (which can help when debugging the software)?

Reply:
Yeah resolved this in my commit 9a5a6b4